TicTacToe.cpp

```diff
@@ -9,16 +9,11 @@ TicTacToe::TicTacToe()
     // use random condition to determine the first player
     if (rand()>(RAND_MAX/2)) {    
         teamSide = 1;       // team blue plays first
-        std::cout << "Blue team plays first." << std::endl; 
```

Comment:
Now that these have been removed, please also take out the `#include <iostream>` line because it's now unused.

Reply:
I made that change just now- let me know if this looks good.